packages/abstract-interface/src/native/ibc_client.rs

```diff
@@ -8,8 +8,4 @@ pub use abstract_core::ibc_client::{
 #[interface(InstantiateMsg, ExecuteMsg, QueryMsg, MigrateMsg)]
 pub struct IbcClient<Chain>;
 
-impl<Chain: CwEnv> IbcClient<Chain> {
-    pub fn new(name: &str, chain: Chain) -> Self {
-        Self(cw_orch::contract::Contract::new(name, chain))
-    }
-}
+impl<Chain: CwEnv> IbcClient<Chain> {}
```

Comment:
cw-orcehstrator update needs this

Reply:
yeah you can just remove the impl alltogether